Setup/UpgradeSchema.php

```diff
@@ -2,16 +2,41 @@
 
 namespace Razorpay\Magento\Setup;
 
+use Razorpay\Magento\Model\Config;
 use Magento\Framework\Setup\UpgradeSchemaInterface;
 use Magento\Framework\Setup\ModuleContextInterface;
+use Razorpay\Magento\Model\TrackPluginInstrumentation;
 use Magento\Framework\Setup\SchemaSetupInterface;
+use Magento\Framework\Module\ModuleListInterface;
+use \Psr\Log\LoggerInterface;
 
-class UpgradeSchema implements  UpgradeSchemaInterface
+class UpgradeSchema implements UpgradeSchemaInterface
 {
-	public function upgrade(SchemaSetupInterface $setup,
-							ModuleContextInterface $context
-						)
+    protected $config;
+    protected $trackPluginInstrumentation;
+    protected $moduleList;
+    protected $logger;
+
+    public function __construct(
+        Config $config,
+        TrackPluginInstrumentation $trackPluginInstrumentation,
+        ModuleListInterface $moduleList,
+        LoggerInterface $logger
+    )
+    {
+        $this->config                       = $config;
+        $this->trackPluginInstrumentation   = $trackPluginInstrumentation;
+        $this->moduleList                   = $moduleList;
```

Comment:
@nikhilsharma1100 `moduleList` is unused within this file.

Reply:
removed